### PR TITLE
Upgrade agent to use active span mechanism available in opentracing-j…

### PR DESCRIPTION
**Pull request by objectiser**
…ava 0.30.0 and focus rules on installing framework integrations, rather than direct instrumentation (i.e. removing the java-net rules).

### CODE REVIEW

**Comment by objectiser** [comment]
Note: will release next version as 0.1.0 as breaking API changes, both in OpenTracing 0.30.0 and as a consequence in the agent's _OpenTracingHelper_ class.
* Decided to remove the previous span management methods on helper class, as not possible to meaningfully deprecate and maintain same behaviour.
* java-net rules have been disabled (will remove at a later date), and removed from the README, as focus of agent from now on should be just installation of framework integrations and custom rules for instrumenting business logic.

File: rules/pom.xml [hunk 1]
```diff
@@ -11,7 +11,11 @@
   <packaging>pom</packaging>
 
   <modules>
+    <!--
```
Comment (pavolloffay) [inline rules/pom.xml]: I would be keener on removing it right away to avoid maintaining of a dead code, or keep it here and mark as deprecated.

**Review by pavolloffay — COMMENTED** [review]
LGTM, although I would no care too much about back compatibility as this is `0.0.x`, rather make significant changes/simplifications now before OT 1.0 is out.

File: rules/pom.xml (hunk 1 above)
Comment (objectiser) [reply rules/pom.xml]: I'll remove and then merge. Thanks!

**Pull request merged**